src/dmd/builtin.d

```diff
@@ -1,4 +1,7 @@
 /**
+ * Evaluate certain external functions for CTFE.
```

Comment:
```suggestion
 * Implement CTFE for intrinsic (builtin) functions
```
Also the empty line to separate title from description.
You might want to link / read https://en.wikipedia.org/wiki/Intrinsic_function